### DISCUSSION TITLE
Create domain

### DISCUSSION THREAD

**Discussion by TechKiran151** [learndevopsonline/learndevopsonline#1566] [Q&A]
While creating domain which shows rdevopsb72.online is already taken can you please suggest which one I need to select?
![Screenshot (453)](https://github.com/learndevopsonline/learndevopsonline/assets/149785664/a57d5b6d-ee8c-4f3f-adb9-bc82926a4b30)

**Comment by sreedharm07** [comment]
Give your own name

**Reply by TechKiran151** [reply]
ok

**Comment by TechKiran151** [comment]
Yes it's resolved.
